python/doc/theory/data_analysis/gaussian_calibration.rst

```diff
@@ -6,6 +6,16 @@ Gaussian calibration
 Introduction
 ~~~~~~~~~~~~
 
+In this page, we present the method used in the :class:`~openturns.GaussianLinearCalibration`
+and :class:`~openturns.GaussianNonLinearCalibration` classes.
+The Gaussian calibration is a particular case of Bayesian calibration 
```

Comment:
Link to a previous suggestion which has been buried by GitHub: https://github.com/openturns/openturns/pull/2060/files/0f6389103932184d73483598cef79791d4abfa60#r1165915860